### PR TITLE
Installation assistant improvements 4th iteration

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|#1481|

## Description

This PR improves the log messages of the installation assistant and removes unused dependencies.

### CODE REVIEW

File: unattended_installer/install_functions/checks.sh [hunk 1]
```diff
@@ -278,21 +278,21 @@ function checks_previousCertificate() {
     fi
 
     if [ -n "${indxname}" ]; then
-        if ! $(tar -tf "${tar_file}" | grep -q "${indxname}".pem) || ! $(tar -tf "${tar_file}" | grep -q "${indxname}"-key.pem); then
+        if ! $(tar -tf wazuh-install-files.tar wazuh-install-files/${indxname}.pem >/dev/null 2>&1) || ! $(tar -tf wazuh-install-files.tar wazuh-install-files/${indxname}-key.pem >/dev/null 2>&1); then
```
Comment (alberpilot) [inline unattended_installer/install_functions/checks.sh:281]: Should we use the`"` for `${indxname}` variable?

File: unattended_installer/install_functions/checks.sh [hunk 2]
```diff
@@ -278,21 +278,21 @@ function checks_previousCertificate() {
...
             common_logger -e "There is no certificate for the indexer node ${indxname} in ${tar_file}."
             exit 1
         fi
     fi
 
     if [ -n "${dashname}" ]; then
-        if ! $(tar -tf "${tar_file}" | grep -q "${dashname}".pem) || ! $(tar -tf "${tar_file}" | grep -q "${dashname}"-key.pem); then
+        if ! $(tar -tf wazuh-install-files.tar wazuh-install-files/${dashname}.pem >/dev/null 2>&1) || ! $(tar -tf wazuh-install-files.tar wazuh-install-files/${dashname}-key.pem >/dev/null 2>&1); then
```
Comment (alberpilot) [inline unattended_installer/install_functions/checks.sh:288]: Same

File: unattended_installer/install_functions/checks.sh [hunk 3]
```diff
@@ -278,21 +278,21 @@ function checks_previousCertificate() {
...
             common_logger -e "There is no certificate for the Wazuh Dashboard node ${dashname} in ${tar_file}."
             exit 1
         fi
     fi
 
     if [ -n "${winame}" ]; then
-        if ! $(tar -tf "${tar_file}" | grep -q "${winame}".pem) || ! $(tar -tf "${tar_file}" | grep -q "${winame}"-key.pem); then
+        if ! $(tar -tf wazuh-install-files.tar wazuh-install-files/${winame}.pem >/dev/null 2>&1) || ! $(tar -tf wazuh-install-files.tar wazuh-install-files/${winame}-key.pem >/dev/null 2>&1); then
```
Comment (alberpilot) [inline unattended_installer/install_functions/checks.sh:295]: Same

File: unattended_installer/install_functions/indexer.sh [hunk 4]
```diff
@@ -161,17 +161,21 @@ function indexer_install() {
 function indexer_startCluster() {
 
     retries=0    
+    set -x
```
Comment (alberpilot) [inline unattended_installer/install_functions/indexer.sh:164]: Remove this

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review

**Review by okynos — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]

File: unattended_installer/install_functions/checks.sh (hunk 1 above)
Comment (DFolchA) [reply unattended_installer/install_functions/checks.sh]: Done in [c07a505](https://github.com/wazuh/wazuh-packages/pull/1488/commits/c07a5050dae2ca4194318489caea40608d04e38a)

File: unattended_installer/install_functions/checks.sh (hunk 2 above)
Comment (DFolchA) [reply unattended_installer/install_functions/checks.sh]: Done in [c07a505](https://github.com/wazuh/wazuh-packages/pull/1488/commits/c07a5050dae2ca4194318489caea40608d04e38a)

File: unattended_installer/install_functions/checks.sh (hunk 3 above)
Comment (DFolchA) [reply unattended_installer/install_functions/checks.sh]: Done in [c07a505](https://github.com/wazuh/wazuh-packages/pull/1488/commits/c07a5050dae2ca4194318489caea40608d04e38a)

File: unattended_installer/install_functions/indexer.sh (hunk 4 above)
Comment (DFolchA) [reply unattended_installer/install_functions/indexer.sh]: Done in [c07a505](https://github.com/wazuh/wazuh-packages/pull/1488/commits/c07a5050dae2ca4194318489caea40608d04e38a)

File: unattended_installer/install_functions/indexer.sh [hunk 5]
```diff
@@ -162,16 +162,19 @@ function indexer_startCluster() {
 
     retries=0    
     for ip_to_test in "${indexer_node_ips[@]}"; do
-        until eval "nc -z ${ip_to_test} 9300" || [ "${retries}" -eq 12 ]; do
+        eval "curl -XGET https://172.16.1.19:9300/ -k -s -o /dev/null"
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply unattended_installer/install_functions/indexer.sh]: Done in [b796094](https://github.com/wazuh/wazuh-packages/pull/1488/commits/b796094b602b7868591ce30485273a3c90f7bb4b)

File: unattended_installer/install_functions/indexer.sh [hunk 6]
```diff
@@ -162,16 +162,19 @@ function indexer_startCluster() {
...
+        e_code="$?"
+        until [ "${e_code}" -ne 7 ] || [ "${retries}" -eq 12 ]; do
             sleep 10
             retries=$((retries+1))
+            eval "curl -XGET https://172.16.1.19:9300/ -k -s -o /dev/null"
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply unattended_installer/install_functions/indexer.sh]: Done in [b796094](https://github.com/wazuh/wazuh-packages/pull/1488/commits/b796094b602b7868591ce30485273a3c90f7bb4b)

**Pull request merged**